PSReadLine/Movement.vi.cs

```diff
@@ -152,7 +152,7 @@ public static void ViEndOfPreviousGlob(ConsoleKeyInfo? key = null, object arg =
         /// <summary>
         /// Returns 0 if the cursor is allowed to go past the last character in the line, -1 otherwise.
         /// </summary>
-        /// <seealso cref="ForwardChar"/>
+        /// <see also cref="ForwardChar"/>
```

Comment:
```suggestion
        /// <seealso cref="ForwardChar"/>
```
`<seealso> is a legit XML doc element`. See https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#seealso